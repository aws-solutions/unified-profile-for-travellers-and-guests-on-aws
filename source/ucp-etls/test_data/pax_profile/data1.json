{    "modelVersion": "1.0",    "id": "8631378892",    "lastUpdatedOn": "2023-03-18T16:39:01.230401322Z",    "createdOn": "2023-02-18T16:39:01.230401322Z",    "lastUpdatedBy": "Dulce Altenwerth",    "createdBy": "Erik Bashirian",    "emails": [        {            "type": "",            "address": "majorrice@kshlerin.org",            "primary": true        },        {            "type": "",            "address": "israellarkin@feeney.com",            "primary": false        },        {            "type": "",            "address": "donnyboyle@ratke.org",            "primary": true        }    ],    "phones": [],    "addresses": [        {            "type": "",            "line1": "876 North Garden port",            "line2": "more content line 2",            "line3": "more content line 3",            "line4": "more content line 4",            "city": "North Las Vegas",            "stateProvince": {                "code": "ND",                "name": ""            },            "postalCode": "63895",            "country": {                "code": "RE",                "name": ""            },            "primary": true        }    ],    "honorific": "Dr",    "firstName": "Ari",    "middleName": "Logan",    "lastName": "Hahn",    "gender": "other",    "pronoun": "he",    "dateOfBirth": "1995-08-26",    "language": {        "code": "mr",        "name": ""    },    "nationality": {        "code": "EG",        "name": ""    },    "jobTitle": "Producer",    "parentCompany": "Overture Technologies",    "loyaltyPrograms": [],    "identityProofs": null}