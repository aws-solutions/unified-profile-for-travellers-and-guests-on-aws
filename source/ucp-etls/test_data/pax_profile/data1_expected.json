{"air_profile_recs": [{"object_type": "pax_profile", "model_version": "1.0", "id": "0049074682", "last_updated": "2022-01-24T04:42:27.919547161Z", "created_on": "2021-09-24T04:42:27.919547161Z", "last_updated_by": "Thaddeus Gusikowski", "created_by": "Garrett Lemke", "honorific": "Sir", "first_name": "Herman", "middle_name": "Aglae", "last_name": "Bergnaum", "gender": "other", "pronoun": "they", "date_of_birth": "1960-12-29", "job_title": "Technician", "company": "SnapSense", "nationality_code": "SZ", "nationality_name": "", "language_code": "ur", "language_name": "", "email_business": "", "email": "mathewcorwin@walter.info", "email_type": "", "phone": "", "phone_home": "", "phone_mobile": "", "phone_business": "1389184526", "phone_type": "business", "traveller_id": "0049074682", "last_updated_partition": "2022/01/24/04/42/27/919547161Z", "address_line1": "", "address_line2": "", "address_line3": "", "address_line4": "", "address_city": "", "address_state_province": "", "address_postal_code": "", "address_country": "", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_postal_code": "", "address_billing_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": ""}], "common_email_recs": [{"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-01-24T04:42:27.919547161Z", "last_updated_by": "Thaddeus Gusikowski", "address": "travisdurgan@sanford.name", "type": "", "traveller_id": "0049074682", "last_updated_partition": "2022/01/24/04/42/27/919547161Z"}, {"model_version": "1.0", "object_type": "email_history", "last_updated": "2022-01-24T04:42:27.919547161Z", "last_updated_by": "Thaddeus Gusikowski", "address": "mathewcorwin@walter.info", "type": "", "traveller_id": "0049074682", "last_updated_partition": "2022/01/24/04/42/27/919547161Z"}], "common_phone_recs": [{"model_version": "1.0", "object_type": "phone_history", "last_updated": "2022-01-24T04:42:27.919547161Z", "last_updated_by": "Thaddeus Gusikowski", "number": "465.725.7891", "country_code": 90, "type": "business", "traveller_id": "0049074682", "last_updated_partition": "2022/01/24/04/42/27/919547161Z"}, {"model_version": "1.0", "object_type": "phone_history", "last_updated": "2022-01-24T04:42:27.919547161Z", "last_updated_by": "Thaddeus Gusikowski", "number": "1389184526", "country_code": 85, "type": "business", "traveller_id": "0049074682", "last_updated_partition": "2022/01/24/04/42/27/919547161Z"}], "air_loyalty_recs": [{"model_version": "1.0", "object_type": "air_loyalty", "last_updated": "2022-01-24T04:42:27.919547161Z", "last_updated_by": "Thaddeus Gusikowski", "id": "2697555871", "program_name": "elite", "miles": 292878.0, "miles_to_next_level": 6671.0, "level": "platinium", "joined": "2021-09-24T04:42:27.919547161Z", "traveller_id": "0049074682", "last_updated_partition": "2022/01/24/04/42/27/919547161Z"}]}